eos_data_distribution/soma_subscription_fetcher.py

```diff
@@ -86,11 +86,9 @@ def _on_interest(self, o, prefix, interest, face, interestFilterId, filter):
             assert filename == 'manifest.json'
             self._subproducers[key] = manifest.Producer(
                 chunkless_name, "%s/v1/%s/manifest.json" % (get_soma_server(), subscription_id), face=face)
-            self._subproducers[key].start()
```

Comment:
I’d keep these `start()` calls around. They’re not doing any harm (an `info` message each), and it’s likely we’ll forget to re-add them when the `start()` implementations are changed to do something again.